### ISSUE TITLE
Block comments do not work inside a table

### ISSUE THREAD

**Issue by rockyallen** [asciidoctor/asciidoctor#1681]
Given this document:

````
= Title

== Section 1
|====
|Col 1
|A
////
|B
////
|C
|==== 
````
B should be excluded, but the B and the block comment markers appear in the output. This is also the behaviour of Asciidoc Python, but should be corrected for Asciidoctor.

**Comment by mojavelinux** [comment]
It should be noted that line comments work in this context, so it's reasonable to assume that block comments should work as well.

```
|====
|Col 1
|A
//|B
|C
|====
```

**Comment by mojavelinux** [comment]
Actually, the problem here is that block comments are technically a delimited block. Therefore, the only way that they are matched is if used in an AsciiDoc table cell. A comment block is found just like any other delimited block, only it is not added to the AST.

It's not out of the question to implement this, but there are some semantic implications.

Here's the line of code where this would need to be done:

https://github.com/asciidoctor/asciidoctor/blob/76dd3f2a982ab21d4b7cd47771fa946de4e21db3/lib/asciidoctor/parser.rb#L861

**Comment by Hollerweger** [comment]
Is a solution still considered?

**Comment by mojavelinux** [comment]
No, it will be addressed by the AsciiDoc Language specification. Once that is complete, we will look into how to implement the requirements in Asciidoctor.
